kip-002.md

```diff
@@ -0,0 +1,52 @@
+
+Author: Keefer Taylor, Hover Labs <keefer@hover.engineering>"
+Created: 04/08/2021
+State: Draft
+
+### Summary
+
+Sets economic parameters in which we'd consider lowering the stability fee based on economic data.
+
+### Motivation
+
+Kolibri's `stability fee` (the rate charged to borrowers) is currently 22%. While this has helped keep peg stable, the high rate may be hindering the growth and adoption of Kolibri. 
+
+In the early days of `kUSD` there was nothing to do with the asset besides sell it for `XTZ`. This effectively meant `kUSD` minters could take long positions on the price of `XTZ`. The other use case available to minters were peer to peer transactions of `kUSD` and holding `kUSD` to liquidate ovens. Demand for long positions greatly outstripped other use cases, and [KIP006](https://forum.tezosagora.org/t/kolibri-governance-proposal-6-continuous-increase-of-the-stability-fee-to-reach-peg/2851) continuously raised the stability fee until peg was achieved. 
+
+Since the early days, several new use cases for kUSD have `emerged`:
+- [Liquidity Providing on Quipuswap](https://analytics.quipuswap.com/pairs/KT1K4EwTpbvYN9agJdjpyJm4ZZdhpUNKB3F6): Earn fees by providing liquidity; Currently ~$1.3M `kUSD` are locked
+- [Kolibri Liquidity Pool](https://kolibri.finance/liquidity-pool): Pool money to help liquidate undercollateralized ovens; Currently ~$680k `kUSD` are locked
+- [Kolibri Farms](https://kolibri.finance/farming): Stake your `kUSD` to receive `kDAO`; currently +$1.5M `kUSD` locked
+- [Crunchy Farms](https://app.crunchy.network/#/farms): Stake your `kUSD` to receive `crDAO`; currently ~$340k `kUSD` locked
+- [Rocket Launchpad](https://twitter.com/Rocketlaunchpd/status/1422394040839024641?s=20): Will accept `kUSD` in their upcoming token sale
+
+We also anticipate new products coming online, specifically:
+- [sexp](https://gitlab.com/smartcontractlabs/sexp-binary-options/-/blob/master/README.md): Trustless binary options settled in `kUSD`
+- Additional Decentralized Exchanges, notably SpicySwap and Plenty Finance, where users can provie liquidity
+- Lending platforms that will provide return on lending kUSD (Specifically [Tezos.Finance](https://tezos.finance/) and [Yupana](https://yupana.finance/) and [Hera](https://docs.heranetwork.co/#2-2-lending)). Hover Labs has also stated that we may want to launch a `kUSD` lending platform
+- Upon activation of the Tezos Granada Protocol (~48 hours from now), Kolibri farms will return double with the increased block time. 
+
+These additional use cases give folks reasons to have reason to want to balance out the adoption of `kUSD`, however, the high stability fee may be limiting demand for these use cases.
+
+### Details
+
+As in [KIP006](https://forum.tezosagora.org/t/kolibri-governance-proposal-6-continuous-increase-of-the-stability-fee-to-reach-peg/2851), we propose lowering the stability fee according to an exponentially weighted moving average of the last 10 days. 
```

Comment:
Small correction, it was EMA10(hours) not EMA10(days)

Reply:
fixed!